src/App.js

```diff
@@ -0,0 +1,33 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import AddCar from './pages/AddCar';
+import DeleteCar from './pages/DeleteCar';
+import Models from './pages/Models';
+import MyReservations from './pages/MyReservations';
+import ReserveCar from './pages/ReserveCar';
+import SignUp from './Components/Authentication/SignUp';
+import LogIn from './Components/Authentication/LogIn';
+import PrivateRoutes from './utils/PrivateRoutes';
+import DetailsPage from './pages/DetailsPage';
+
+function App() {
```

Comment:
- [ ] When I add a new car, I am presented with the "username" field.  Maybe it should be car name?

![image](https://github.com/yodit93/car-reservation-front-end/assets/57421823/c1dd0871-11c1-4896-9e8b-19a000264b29)

![image](https://github.com/yodit93/car-reservation-front-end/assets/57421823/4c42b388-cc0f-4ced-998e-e939719d47cf)

I end up with a car that is named as my username.


Please, rename that field in your form so it is more consistent (like car name). 👍

Reply:
Hello @Meltrust, thanks for the observation, the file has been updated.